protoc-gen-openapiv2/internal/genopenapi/template_test.go

```diff
@@ -10498,6 +10498,247 @@ func TestArrayMessageItemsType(t *testing.T) {
 	}
 }
 
+func TestArrayMessageItemsTypeOmitWhenRefSibling(t *testing.T) {
```

Comment:
almost the same test as [TestArrayMessageItemsType](https://github.com/jgreffe/grpc-gateway/blob/f8c135ede12a9540475d1d5459ff325a95dc3b3a/protoc-gen-openapiv2/internal/genopenapi/template_test.go#L10262), but we check we don't have results like [these](https://github.com/jgreffe/grpc-gateway/blob/f8c135ede12a9540475d1d5459ff325a95dc3b3a/protoc-gen-openapiv2/internal/genopenapi/template_test.go#L10414-L10415).